### ISSUE TITLE
End lines of French footnotes hidden

### ISSUE THREAD

**Issue by josephwright** [josephwright/beamer#95]
Hello,

as pointed by the following MCE, end lines of French footnotes are hidden with sidebar themes.

Thanks!

````
\documentclass[french]{beamer}
\usepackage{babel}
\usetheme{PaloAlto}
\begin{document}
  \begin{frame}
    test\footnote{Lorem ipsum dolor sit amet, consectetuer adipiscing
      elit. Ut purus elit, vestibulum ut, placerat ac, adipiscing
      vitae, felis.}
  \end{frame}
\end{document}

````

----------------------------------------
- Bitbucket: https://bitbucket.org/rivanvx/beamer/issue/95
- Originally reported by: denisbitouze
- Originally created at: 2010-11-08

**Comment by josephwright** [comment]
_From Vedran Miletić on 2011-01-03_

Interesting, it is really French-specific.

Joseph, can you look into it?

**Comment by josephwright** [comment]
Okay, I've 'fixed' this in 1f24e84cbb9d9359801c33e38ec702b56bdc4b06 by simply ignoring the changes that `babel` makes: it was using it's own code for footnotes. I wonder if French people need a better solution in the long-term, but this at least deals with the out-and-out bug.

**Comment by josephwright** [comment]
_From denisbitouze on 2011-01-17_

Yes, we need a better solution at least for two reasons:

    French typography commands that footnotes are typeset in a peculiar way: the numbers in the footnotes are not superscript, are (by default) in normal font and size, and are followed by a period followed by a space. For instance: 
````
\documentclass[french]{article}
\usepackage{babel}
\begin{document}
  \begin{frame}
    Foo\footnote{Bar.}
  \end{frame}
\end{document}
````
    The current solution doesn't work :) as pointed out by the following MCE (tested with a clone repository): 
````
\documentclass[french]{beamer}
\usepackage{babel}
\usetheme{PaloAlto}
\begin{document}
  \begin{frame}
    test\footnote<.->{Bien entendu, ce ne sont pas les
      \emph{utilisateurs} des traitements de texte qui le sont...}
  \end{frame}
\end{document}
````
Thanks!

**Comment by josephwright** [comment]
_From Joseph Wright on 2011-01-20_

There are actually two different things going on here. The French typography question is one, but your second example fails even without `babel`:
````
\documentclass{beamer}
\usetheme{PaloAlto}
\begin{document}
  \begin{frame}
    test\footnote{Bien entendu, ce ne sont pas les
      utilisateurs des traitements de texte qui le sont...}
  \end{frame}
\end{document}
````

**Comment by josephwright** [comment]
_From denisbitouze on 2011-09-18_

Hi,

I make this issue reborn :)

Firstly, the trouble still occurs, that is end lines of footnotes are hidden with sidebar themes
````
\documentclass{beamer}
\usetheme{PaloAlto}
\begin{document}
  \begin{frame}
    test\footnote{Bien entendu, ce ne sont pas les
      utilisateurs des traitements de texte qui le sont...}
  \end{frame}
\end{document}
````
But, moreover, maybe because of this in #86ed42e273fa86ed42e273fa, French footnotes are not typeset like they should, as pointed out by present comment #4.

Thanks!

**Comment by josephwright** [comment]
Fixed by 212b20e6c3a9639e20c601fcaa71d5f67cecffdf

**Comment by josephwright** [comment]
_From denisbitouze on 2012-03-12_

It works now like a charm. Many Thanks!
